test/optim/test_utils.py

```diff
@@ -153,7 +169,7 @@ def test_get_extra_mll_args(self):
 
 class TestExpandBounds(BotorchTestCase):
     def test_expand_bounds(self):
-        X = torch.zeros(2, 3)
+        X = torch.zeros(4, 2, 3)
```

Comment:
we should probably test with and without explicit t-batch dimension

Reply:
Just to clarify, do we want to test without explicit t-batch when `bounds` is <=2 dim, or even when `bounds` has a t-batch? If it's the second case, I am assuming this should raise an error if `bounds` has a t-batch and `X` doesn't.